src/server.tsx

```diff
@@ -91,25 +91,40 @@ const [, fetchConfiguredArticleEpicTestsCached] = cacheAsync(
     `fetchConfiguredEpicTests_ARTICLE`,
 );
 
+const [, fetchConfiguredArticleEpicHoldbackTestsCached] = cacheAsync(
+    () => fetchConfiguredEpicTests('ARTICLE_HOLDBACK'),
+    60,
+    `fetchConfiguredEpicTests_ARTICLE_HOLDBACK`,
+);
+
 const [, fetchConfiguredLiveblogEpicTestsCached] = cacheAsync(
     () => fetchConfiguredEpicTests('LIVEBLOG'),
     60,
     `fetchConfiguredEpicTests_LIVEBLOG`,
 );
 
-const fetchCachedEpicTests: { [key in EpicType]: () => Promise<EpicTests> } = {
```

Comment:
I don't think we need this object as we're not really doing any dynamic looks ups e.g.

```
const fetchTests = fetchCachedEpicTests[epicType];
```